demo-simple/src/main/res/layout/activity_main.xml

```diff
@@ -125,17 +125,9 @@
     android:text="Delete all" />
 
   <Button
-    android:id="@+id/button_show_completed_downloads"
+    android:id="@+id/btn_log_files_dir"
```

Comment:
Avoid abbreviations whenever possible and please respect the naming convention already in the project, it avoids confusion and unnecessary maintenance down the line.